lib/src/Uncertainty/Algorithm/Transformation/ProcessTransformation/BoxCoxFactory.cxx

```diff
@@ -381,6 +484,7 @@ BoxCoxTransform BoxCoxFactory::build(const Sample & inputSample,
                                      const Point & shift,
                                      GeneralLinearModelResult & result)
 {
+  LOGWARN(OSS() << "BoxCoxFactory::build(inputSample, outpuSample, covarianceModel, basisCollection, shift) is deprecated");
```

Comment:
can you update the changelog ?
maybe you could also send the PR that removes the method in master

Reply:
1st point ok (already done)
For the second point I will have a specific PR for both GLM/KA & BoxCoxFactory